### PR TITLE
Source document retrieval and persistence in Content Service

### PR DESCRIPTION
**Pull request by Rossall82**
- Persist status of source doc retrieval in db
- Clear source doc from queue once retrieved
- Retrieve Content Service creds from AKV
- Store retrieved doc in Content Service
- Add API method to clear source doc request queue (Swagger use only)

### CODE REVIEW

File: src/SourceDocumentCoordinator.IntegrationTests/SourceDocumentCoordinatorIntegrationTests.cs [hunk 1]
```diff
@@ -35,6 +42,22 @@ public async Task Test_DataServiceApi_Connectivity()
...
+            {
+                Credentials = new NetworkCredential(_secretsConfigWrapper.Value.ContentServiceUsername,
+                    _secretsConfigWrapper.Value.ContentServicePassword,
+                    _secretsConfigWrapper.Value.ContentServiceDomain)
+            };
+
+            var httpClient = new HttpClient(hndlr);
+            _contentServiceApiClient = new ContentServiceApiClient(httpClient, _uriConfigWrapper, _secretsConfigWrapper);
+
+            await _contentServiceApiClient.Post(System.IO.File.ReadAllBytes("C:\\temp\\2.pdf"), "2.pdf");
```
Comment (UKHOGreg) [inline src/SourceDocumentCoordinator.IntegrationTests/SourceDocumentCoordinatorIntegrationTests.cs]: Temp PDF path probably needs replacing

File: src/SourceDocumentCoordinator/SourceDocumentCoordinator/HttpClients/ContentServiceApiClient.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                default:
+                    throw new NotImplementedException();
+            }
+        }
+
+        private static ContentServicePostRequest ConstructContentServicePostRequest()
+        {
+            var metadata = new ContentServiceMetadata
+            {
+                ProductType = "SDRASourceDocument", // TODO: Move to Azure app config
```
Comment (UKHOGreg) [inline src/SourceDocumentCoordinator/SourceDocumentCoordinator/HttpClients/ContentServiceApiClient.cs]: As the comment says, lets move this to Azure App Configuration!

File: src/SourceDocumentCoordinator.IntegrationTests/SourceDocumentCoordinatorIntegrationTests.cs (hunk 1 above)
Comment (Rossall82) [reply src/SourceDocumentCoordinator.IntegrationTests/SourceDocumentCoordinatorIntegrationTests.cs]: Complete!

File: src/SourceDocumentCoordinator/SourceDocumentCoordinator/HttpClients/ContentServiceApiClient.cs (hunk 2 above)
Comment (Rossall82) [reply src/SourceDocumentCoordinator/SourceDocumentCoordinator/HttpClients/ContentServiceApiClient.cs]: Can't be bothered

**Comment by gareth72evans** [comment]
Top Banana!

**Pull request merged**